src/html5/parser/document.rs

```diff
@@ -16,6 +17,11 @@ use std::collections::HashMap;
 use std::fmt::Display;
 use std::ops::{Deref, DerefMut};
 
+extern crate queues;
+use queues::*;
+
+use super::query::SearchType;
```

Comment:
this was auto imported by Zed... reminder for myself to change this to `use crate::...` instead